docs/2.7.0/docs/canton/usermanual/troubleshooting_guide.rst

```diff
@@ -134,54 +134,39 @@ Log Files
+     In above example, we see the trace id twice: ``tid:d5df95972a95b5ff00cb5cc3346c545f`` and ``NOT_SEQUENCED_TIMEOUT(2,d5df9597)``. By filtering according to the ``trace-id``, you can find almost all log statements that relate to a particular command. The ledger-api server logs the ``trace-id`` as well. However, sometimes, we also need to find out the command id of a transaction. You can do that by grepping for the “rosetta stone”, which is one particular log line that contains both strings::
 
-          2022-10-19 17:45:20,630 [⋮] DEBUG 
-          c.d.c.p.s.CantonSyncService:SimplestPingIntegrationTestInMemory/participant=participant1 
+          2022-10-19 17:45:20,630 [⋮] DEBUG
+          c.d.c.p.s.CantonSyncService:SimplestPingIntegrationTestInMemory/participant=participant1
           tid:d5df95972a95b5ff00cb5cc3346c545f - Received submit-transaction
-          5ec48477-b440-4175-a1f8-d8373d5108ef-ping-5838bf58-5e1d-4ccc-869b-0607bed42fc7 
+          5ec48477-b440-4175-a1f8-d8373d5108ef-ping-5838bf58-5e1d-4ccc-869b-0607bed42fc7 from ledger-api server
 
      The first string is again the trace id and the second one is the command id. Ideally, you then subsequently filter for command-id and trace-id.
 
 * **Extract the Context of a Log Message**
 
-     The log lines often also contain the “context” of the component. Examples: 
+     The log lines often also contain the “context” of the component. Examples:
 
      * This log line tells us which component of which participant (participant1) of which domain connection (da) has been emitting this log line. It also includes the trace id of the underlying request::
-          
-          2022-10-04 15:55:50,077 [⋮] DEBUG 
-          c.d.c.p.p.TransactionProcessingSteps:participant=participant1/domain=da 
-          tid:461cae6245cfaadc87c2481a17d7e1bb - Preparing batch for transaction 
+
+          2022-10-04 15:55:50,077 [⋮] DEBUG
+          c.d.c.p.p.TransactionProcessingSteps:participant=participant1/domain=da
+          tid:461cae6245cfaadc87c2481a17d7e1bb - Preparing batch for transaction
           submission
 
      * During tests, the log line includes the name of the test. In this case, it is ``SimplestPingIntegrationTestInMemory``::
 
           :: 2022-10-04 15:55:50,077 [⋮] DEBUG c.d.c.p.p.TransactionProcessingSteps:SimplestPingIntegrationTestInMemory/
-          participant=participant1/domain=da tid:461cae6245cfaadc87c2481a17d7e1bb 
+          participant=participant1/domain=da tid:461cae6245cfaadc87c2481a17d7e1bb
           - Preparing batch for transaction submission
 
 
-     * Some classes use a different format for logging the context. This time, the context comes after the log message::
```

Comment:
I think 3rd parties may still appear with a different format. Let me check that

Reply:
Well, drop that. Actually 3rd party logs look box standard. They're just less verbose
```
2023-07-05 09:10:59,843 [⋮] INFO  a.e.s.Slf4jLogger - Slf4jLogger started
```